lib/components/narrative/narrative-itineraries.js

```diff
@@ -444,8 +445,12 @@ class NarrativeItineraries extends Component {
           itinerary: cur
         })
 
+        const configModeOverrides = Object.values(routeModeOverrides)
```

Comment:
This line is making Percy tests fail. At least pass `{}` if `routeModeOverrides` is not set.